marblerun-coordinator/templates/marble-injector.yaml

```diff
@@ -55,6 +65,13 @@ spec:
         ports:
           - containerPort: 8443
             name: http
+        resources:
+          requests:
+            memory: "5Mi"
```

Comment:
How did you get to those values?

Reply:
Some manual testing and benchmarking on minikube with the metrics server plugin enabled.
Maybe it is not necessary to set any values here, since the resource requirements are expected to stay small.
However we should not just set a limit, since only specifying a limit results in kubernetes automatically setting resource requests equal to the chosen limit.